src/main/java/task/NewOpportunityPortal/service/impl/MessageServiceImpl.java

```diff
@@ -33,27 +34,23 @@ public Long createMessage(Message message) {
         try {
             log.info("Encoder message");
             message.setText(encryptDecrypt.encrypt(message.getText()));
-        } catch (BadPaddingException | IllegalBlockSizeException e) {
-            log.error(e.getMessage());
-        } catch (NoSuchAlgorithmException | InvalidKeyException | NoSuchPaddingException | UnsupportedEncodingException e) {
-            e.printStackTrace();
+        } catch (BadPaddingException | IllegalBlockSizeException | NoSuchAlgorithmException | NoSuchPaddingException | InvalidKeyException | UnsupportedEncodingException e) {
+            log.error(Arrays.toString(e.getStackTrace()));
```

Comment:
you may just type `log.error(e)` or `log.error("Error during text encryption: ", e)`